packages/compass-serverstats/src/stores/current-op-store.js

```diff
@@ -87,7 +87,7 @@ const CurrentOpStore = Reflux.createStore({
     let error = null; let response;
 
     try {
-      response = await this.dataService.currentOp(false);
```

Comment:
Only the "currentOp" test was using `true` value, so we can drop the argument and always use `allUsers: true` in the `$currentOp` stage.

Reply:
We don’t even really need the `dataService.currentOp()` helper then, right?

Reply:
Well, we don't. But I do like having this even small logic wrapped into the currentOp method. Do you feel string about removing it?

Reply:
No, no strong feelings either way :slightly_smiling_face: